.github/workflows/jekyll_build_work.yml

```diff
@@ -0,0 +1,34 @@
+name: Jekyll build works
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    name: Jekyll build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-java@v1
+        with:
+          java-version: '1.8'
+      - uses: kamiazya/setup-graphviz@v1
+      - name: Installs libfontconfig
+        run: sudo apt-get install -y libfontconfig
+      - uses: actions/setup-ruby@v1
+        with:
+          ruby-version: '2.6'
+      - name: Install bundle
+        run: |
+          sudo apt-get -yqq install libpq-dev
+          gem install bundler
+      - name: Bundle install
+        run: |
+          bundle install --jobs 4 --retry 3
+      - name: Run build
+        run: bundle exec jekyll build
+      - name: Check for plantUml
+        run: |
+          grep './_site/index.html' -e 'PlantUML version 1.2020.05'
```

Comment:
I realize we need to add something in the output from `kramdown-plantuml` we can search for here that isn't reliant upon the version of PlantUML in use. But this will do for now. 👍

Reply:
I created SwedbankPay/kramdown-plantuml#18 to track this.

Reply:
I agree, but of the output it is the one I find the most prudent to check as each of the lines I want to look at is version dependent. When the kramdown-plantuml package is updated and this test fail that is to be expected since we are testing the output of an external dependency.